modules/manage/pages/disaster-recovery/shadowing/setup.adoc

```diff
@@ -105,28 +105,39 @@ endif::[]
 To set up Shadowing, you need to create a shadow link and configure filters to select which topics, consumer groups, and ACLs to replicate. 
 
 ifdef::env-cloud[]
-You can set up Shadowing with the Redpanda Cloud UI, `rpk`, or the Data Plane API. 
+If using the Cloud API to set up Shadowing, you must link:/api/doc/cloud-controlplane/authentication[authenticate] to the API by including an access token in your requests.
 endif::[]
 
+=== Create a shadow link
+
 ifndef::env-cloud[]
-You can set up Shadowing with Redpanda Console, `rpk`, or the Admin API. 
+Any cluster can create a shadow link to a source cluster. When you create a shadow link with `rpk` or the API, you need to provide a YAML configuration that defines connection settings, filters, and synchronization options.
 endif::[]
 
-== Create a shadow link
-
-Any cluster can create a shadow link to a source cluster. When you create a shadow link with `rpk` or the API, you need to provide a YAML configuration file that defines connection settings, filters, and synchronization options.
+ifdef::env-cloud[]
+Any BYOC or Dedicated cluster can create a shadow link to a source cluster. When you create a shadow link with `rpk` or the API, you need to provide a YAML configuration that defines connection settings, filters, and synchronization options.
+endif::[]
 
 [TIP]
 ====
 You can use `rpk` to generate a sample configuration file with common filter patterns:
 
 [,bash]
 ----
+ifndef::env-cloud[]
 # Generate a sample configuration file with placeholder values
 rpk shadow config generate -o shadow-config.yaml
 
 # Or generate a template with detailed field documentation
 rpk shadow config generate --print-template -o shadow-config-template.yaml
+endif::[]
+ifdef::env-cloud[]
+# Generate a sample configuration file with placeholder values
+rpk shadow config generate --from-cloud -o shadow-config.yaml
+
+# Or generate a template with detailed field documentation
+rpk shadow config generate --from-cloud --print-template -o shadow-config-template.yaml
```

Comment:
Unfortunately, this is not currently supported for Cloud, we will add it for the upcoming releases